ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -123,6 +126,7 @@ descriptor:
   glycol: "Liquid mixture consisting of glycol and water "
   green: "green light fracture of ambient light"
   ground: "The solid surface of the earth."
+  group: "A group of entities or devices"
```

Comment:
would propose a definition that doesnt include itself, such as "a set of entities or devices"